src/activities/tangram/Tangram.qml

```diff
@@ -132,8 +132,8 @@ ActivityBase {
                     sourceSize.width: modelData.width * background.playWidth
                     sourceSize.height: modelData.height * background.playWidth
                     z: index
-                    rotation: modelData.flipping ? 360 - modelData.rotation : modelData.rotation
-                    mirror: modelData.flipping ? true : false
+                    rotation: modelData.rotation
```

Comment:
do you have an example of level where it applies? I'm not sure to understand the change

Reply:
11 levels created this problem with parallelogram. *![screenshot](https://pasteboard.co/Hay4Ont.png)*
The rotation of modelTan was out of place and the solution was being accepted.